src/diffusers/models/vae_flax.py

```diff
@@ -87,6 +109,22 @@ def __call__(self, hidden_states):
 
 
 class FlaxResnetBlock2D(nn.Module):
+    """
+    Flax implementation of 2D Resnet Block.
+
+    Args:
+        in_channels (`int`):
+            Input channels
+        out_channels (`int`):
+            Output channels
+        dropout_prob (:obj:`float`, *optional*, defaults to 0.0):
```

Comment:
I think we should put `dropout` instead of `dropout_prob` for consistency

Reply:
Oh I got it, you cannot have this modification since the line `self.dropout = Dropout(self.dropout_prob)` won't work

Reply:
I attempt to fix it at: https://github.com/huggingface/diffusers/pull/595